features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandler.kt

```diff
@@ -113,7 +113,7 @@ internal class TelemetryEventHandler(
 
         val eventIdentity = event.identity
 
-        if (seenInCurrentSession.contains(eventIdentity)) {
+        if (!event.isMetric && seenInCurrentSession.contains(eventIdentity)) {
```

Comment:
do we have a risk of sending too much metrics by disabling this check and thus adding a lot of pressure on the storage and also on the Datadog servers?

Reply:
I was following what was done in iOS here. I guess the risk was assessed already there, in the end we are going to have an extra sampler in the `MetricsDispatcher` which will come in the next PR. We have a `15%` extra sampler there.